newsfragments/41758.significant.rst

```diff
@@ -10,3 +10,28 @@ Removed deprecated functions and modules from ``airflow.www`` module.
   ``airflow.utils.log.secrets_masker.get_sensitive_variables_fields`` instead.
 - Removed the method ``should_hide_value_for_key()`` from ``airflow.www.utils``: Please use
   ``airflow.utils.log.secrets_masker.should_hide_value_for_key`` instead.
+
+* Types of change
+
+  * [ ] DAG changes
+  * [x] Config changes
+  * [ ] API changes
+  * [ ] CLI changes
+  * [ ] Behaviour changes
+  * [ ] Plugin changes
+  * [ ] Dependency change
+  * [ ] Code interface change
```

Comment:
I would say due to deprecations if people use it in private code it is a non-public API cleanup.
```suggestion
  * [x] Code interface change
```